edgelet/doc/product-info.plantuml

```diff
@@ -0,0 +1,40 @@
+@startuml
```

Comment:
what extension is required to read this?

Reply:
It can be rendered to PNG with [plantuml](https://plantuml.com/). Should I add the rendered diagram?

Reply:
Yes, check in the PNGs and reference them from the markdown docs. Just like https://github.com/Azure/iot-identity-service/tree/main/docs-dev/img and https://github.com/Azure/iot-identity-service/blob/main/docs-dev/Makefile

Reply:
I will add a documentation page for this feature and use the rendered PNG there, in that case. The rendered PNG is present as of 97d0809ca39869c75fc479b29f9be86694798895.

Reply:
It has been decided that documentation will be included in a separate PR.